client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1595,7 +1595,7 @@
   "placeholder.search-by-name-or-code": "Search by name or code",
   "placeholder.search-by-notes": "Search by notes",
   "preferences": "Preferences",
-  "prescription": "Prescription",
+  "prescription": "Prescriptions",
```

Comment:
This is super annoying -- I tried for ages to figure out where the keys were defined in the breadcrumbs, and couldn't figure out how to change it to the plural version. But it turns out all the other areas (Outbound shipments, etc) have just put the plural version in the singular key! 🙄

Not a great pattern, but too hard to fix right now.